### ISSUE TITLE
[Feature] Generate Warning for Multiple Traps on the Same Signal

### ISSUE THREAD

**Issue by fxierh** [koalaman/shellcheck#2920]
#### For new checks and feature suggestions
- [x] https://www.shellcheck.net/ (i.e. the latest commit) currently gives no useful warnings about this
- [x] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/usr/bin/env bash

cleanup() {
    echo "foo"
}

cleanup2() { 
    echo "bar"
}

echo "start"
trap cleanup EXIT
trap cleanup2 EXIT
```

#### Here's what shellcheck currently says:
```sh
$ shellcheck myscript
No issues detected!
```

#### Here's what I wanted or expected to see: 
A warning.
